x-pack/test/observability_functional/apps/observability/alerts/alert_status.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { ALERT_STATUS_RECOVERED, ALERT_STATUS_ACTIVE } from '@kbn/rule-data-utils';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+const ALL_ALERTS = 40;
+const ACTIVE_ALERTS = 10;
+const RECOVERED_ALERTS = 30;
+
+export default ({ getService }: FtrProviderContext) => {
+  const esArchiver = getService('esArchiver');
+
+  describe('alert status filter', function () {
+    this.tags('includeFirefox');
+
+    const observability = getService('observability');
+    const retry = getService('retry');
+
+    before(async () => {
+      await esArchiver.load('x-pack/test/functional/es_archives/observability/alerts');
+      await esArchiver.load('x-pack/test/functional/es_archives/infra/metrics_and_logs');
+      await observability.alerts.common.navigateToTimeWithData();
+    });
+
+    after(async () => {
+      await esArchiver.unload('x-pack/test/functional/es_archives/observability/alerts');
+      await esArchiver.unload('x-pack/test/functional/es_archives/infra/metrics_and_logs');
+    });
+
+    it('is filtered to only show "all" alerts by default', async () => {
+      await retry.try(async () => {
+        const tableRows = await observability.alerts.common.getTableCellsInRows();
+        expect(tableRows.length).to.be(ALL_ALERTS);
+      });
+    });
+
+    it('can be filtered to only show "active" alerts using the filter button', async () => {
+      await observability.alerts.common.setAlertStatusFilter(ALERT_STATUS_ACTIVE);
```

Comment:
@fkanout Initially we started having all services in one big file and then we refactored and split services in sub-services (pagination, bulk_actions, add_to_case etc). I know it is just one helper function related to AlertStatusFilter and it could stay in common.ts file https://github.com/fkanout/kibana/blob/119946-remove-workflow-code-update-tests/x-pack/test/functional/services/observability/alerts/common.ts#L229, but ideally we would like to keep sub-services separate. It can be refactored later, I just wanted to mention the pattern we use.

Reply:
Agree, @mgiota. But this function was already there 😄

Reply:
I don't like navigating in a long file too.  If you prefer, I can put that function in a separate file?